analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -403,31 +403,31 @@ class PIP(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: Repos
         return ProjectAnalyzerResult(project, packages.map { it.toCuratedPackage() }.toSortedSet())
     }
 
-    private fun getBinaryArtifact(pkg: Package, pkgReleases: ArrayNode): RemoteArtifact {
+    private fun getBinaryArtifact(pkg: Package, artifacts: ArrayNode): RemoteArtifact {
         // Prefer python wheels and fall back to the first entry (probably a sdist).
-        val pkgRelease = pkgReleases.asSequence().find {
+        val binaryArtifact = artifacts.asSequence().find {
             it["packagetype"].textValue() == "bdist_wheel"
-        } ?: pkgReleases[0]
+        } ?: artifacts[0]
 
-        val url = pkgRelease["url"]?.textValue() ?: pkg.binaryArtifact.url
-        val hash = pkgRelease["md5_digest"]?.textValue() ?: pkg.binaryArtifact.hash
+        val url = binaryArtifact["url"]?.textValue() ?: pkg.binaryArtifact.url
+        val hash = binaryArtifact["md5_digest"]?.textValue() ?: pkg.binaryArtifact.hash
 
-        return RemoteArtifact(url = url, hash = hash, hashAlgorithm = HashAlgorithm.fromHash(hash))
+        return RemoteArtifact(url, hash, HashAlgorithm.fromHash(hash))
     }
 
-    private fun getSourceArtifact(pkgReleases: ArrayNode): RemoteArtifact {
-        val pkgSources = pkgReleases.asSequence().filter {
+    private fun getSourceArtifact(artifacts: ArrayNode): RemoteArtifact {
```

Comment:
Considering only the those two function the change is fine.
The rename of the second parameter however makes the code style a bit inconsistent with the
local code style, e.g. the caller of `getSourceArtifact` still calls the node `pkgReleases`.
What I like about that is that it somehow indicates what 'JSON' node it expects.

How about (1) using 'releaseNode' as param name and also renaming the variable upwards the call tree?

Another alternative could be to pass the 'parent' node (the package one) to these function - it has the drawback that the function gets more than it needs and the advantage that always the package node could be passed and thus achieve more clarity about which JSON node were dealing with.

Reply:
> The rename of the second parameter however makes the code style a bit inconsistent with the local code style, e.g. the caller of `getSourceArtifact` still calls the node `pkgReleases`.

Correct, that actually was part of the intention. I don't particularly like the variable-naming-style the caller uses, with prefixing all variables with `pkg`. If everything is `pkg` anyway, you may as well drop the prefix.

> What I like about that is that it somehow indicates what 'JSON' node it expects. [...] How about (1) using 'releaseNode' as param name [...]

Yes, that makes sense to me, will do.

> [...] and also renaming the variable upwards the call tree?

I'd prefer not to do that, at least not as part of this change, whose scope is limited to these sibling-functions.

Reply:
I don't like the `pkg` prefix either, my point was mainly consistent style within one file - however your proposed compromise makes sense as well.